<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>831</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>831</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Time Tracker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/hourglass_icon.png</normaloff>:/resources/hourglass_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DateLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTaskButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&amp;Add Task...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/add_button_img.png</normaloff>
      <normalon>:/resources/add_button_img.png</normalon>
      <activeoff>:/resources/add_button_img.png</activeoff>
      <activeon>:/resources/add_button_img.png</activeon>:/resources/add_button_img.png</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="taskTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>801</width>
      <height>331</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <column>
     <property name="text">
      <string>Task Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Task Start</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Task End</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Task Duration</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Notes</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="stopTaskButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&amp;Stop Current Task</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/stop_button_img.png</normaloff>:/resources/stop_button_img.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="totalMinutesLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Minutes Go Here</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalMinutesLabel_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Minutes:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTime_Tracker">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="menuActionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuTime_Tracker"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menuActionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>S&amp;ettings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
